[201811]: security update kernel to 4.9.246

lguohan opened the pull request:
Signed-off-by: Guohan Lu <lguohan@gmail.com>

qiluo-msft commented on patch/preconfig/series:
```diff
@@ -1 +1 @@
-changelog.patch
+# changelog.patch
```
Will you remove this line and this file?

lguohan replied:
not necessary

qiluo-msft reviewed (commented):
As comment

paulmenzel commented:
Thank you for the update.

1.  Please split this into three commits with respective commit messages.
    a)  patch/series: Remove changelog.patch
    b)  Introduce variable for base URL
    c)  Update kernel to 4.9.246 (the tag is not needed in the commit message summary, as it’s in the correct branch)
2.  Also, it’d be great if you mentioned the security issue being fixed in the commit message of the commit updating the Linux version.
3.  Lastly, please mention if this has been tested somewhere.

lguohan commented:
b) and c) is hard to split since base_url also changed.

added changelog for security issues from debian upstream

since it is taken from debian kernel, so we just did the build test.

paulmenzel commented:
> b) and c) is hard to split since base_url also changed.

Next time, first introduce the variable with the “old” URL, and then change the URL.
